providers/fips/fipsprov.c

```diff
@@ -910,6 +912,9 @@ int OSSL_provider_init(const OSSL_PROVIDER *provider,
         case OSSL_FUNC_CORE_POP_ERROR_TO_MARK:
             c_pop_error_to_mark = OSSL_get_core_pop_error_to_mark(in);
             break;
+        case OSSL_FUNC_CORE_VSNPRINTF:
+            c_core_vsnprintf = OSSL_get_core_vsnprintf(in);
+            break;
         case OSSL_FUNC_CRYPTO_MALLOC:
             c_CRYPTO_malloc = OSSL_get_CRYPTO_malloc(in);
             break;
```

Comment:
Note for future cleanup: we should really consider having common provider functions for this sort of thing.  I tried to start that with `providers/common/include/prov/bio.h` and `providers/common/bio_prov.c`